client/web/workflow/src/components/ExpressionTable.vue

```diff
@@ -1,105 +1,142 @@
 <template>
-  <b-table
-    id="expression-table"
-    fixed
-    borderless
-    hover
-    head-row-variant="secondary"
-    details-td-class="bg-white"
-    class="mb-4"
-    :items="items"
-    :fields="fields"
-    thead-tr-class="border-thick"
-    :tbody-tr-class="rowClass"
-    @row-clicked="item => $set(item, '_showDetails', !item._showDetails)"
-  >
-    <template #cell(target)="{ value }">
-      <var>{{ value }}</var>
-    </template>
-
-    <template #cell(type)="{ value }">
-      <var>{{ value }}</var>
-    </template>
-
-    <template #[`cell(${valueField})`]="{ value, item, index }">
+  <div class="table-responsive">
+    <div class="row header pl-3 font-weight-bold">
       <div
-        class="d-flex justify-content-between align-items-center"
+        v-for="(field, index) in fields"
+        :key="index"
+        :class="`col ${field.thClass}`"
+      >
+        {{ field.label }}
+      </div>
+    </div>
+
+    <draggable
+      :list="items"
+      handle=".pointer"
+      @end="$emit('order-updated')"
+    >
+      <div
+        v-for="(item, index) in items"
+        :key="index"
       >
         <div
-          class="text-truncate"
-          :class="{ 'w-75': item._showDetails}"
+          class="d-flex justify-content-between align-items-center pointer expr-item"
+          @click="$set(item, '_showDetails', !item._showDetails)"
         >
-          <samp>{{ value }}</samp>
+          <div
```

Comment:
b-col, loop through the fields and show correct template based on type and formatter.